### PR TITLE
Shooter interpolation and limelight patches

### PR DESCRIPTION
**Pull request by natesales**

### CODE REVIEW

File: src/main/java/org/team1540/robot2022/utils/Limelight.java [hunk 1]
```diff
@@ -56,11 +55,9 @@ public double getCalculatedDistance() {
         return actualHeight / Math.tan(theta);
     }
 
-    /**
-     * Publishes the current calculated distance (in inches) to the SmartDashboard.
-     */
-    public void publishCalculatedDistance() {
-        SmartDashboard.putNumber("limelight/calculatedDistance", 39.37007874 * getCalculatedDistance());
+    public void updateSmartDashboardValues() {
+        SmartDashboard.putNumber("limelight/custom/calculatedDistance", 39.37007874 * getCalculatedDistance());
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/utils/Limelight.java:59]: Maybe add a comment to explain `39.37007874`

File: src/main/java/org/team1540/robot2022/utils/Limelight.java [hunk 2]
```diff
@@ -80,11 +77,13 @@ public Vector2d getTargetAngles() {
...
-        boolean verticalInBounds = y > Math.toRadians(-21);
-        boolean horizontalInBounds = x > Math.toRadians(-22) && x < Math.toRadians(17);
-        return (double) limelightTable.getEntry("tv").getNumber(0) > 0 && verticalInBounds && horizontalInBounds;
+        return getTargetAngles().x != 0;
+
+//        double y = getTargetAngles().y;
+//        double x = getTargetAngles().x;
+//        boolean verticalInBounds = y > Math.toRadians(-21);
+//        boolean horizontalInBounds = x > Math.toRadians(-22) && x < Math.toRadians(17);
+//        return (double) limelightTable.getEntry("tv").getNumber(0) > 0 && verticalInBounds && horizontalInBounds;
```
Comment (rutmanz) [inline src/main/java/org/team1540/robot2022/utils/Limelight.java:86]:
```suggestion
```
If this is unused, it should probably just be removed

**Review by rutmanz — COMMENTED** [review]
Maybe we should make `InterpolationTable` a singleton. It's pretty annoying to have to pass it everywhere

**Review by sanashah007 — APPROVED** [review]

File: src/main/java/org/team1540/robot2022/utils/Limelight.java (hunk 1 above)
Comment (natesales) [reply src/main/java/org/team1540/robot2022/utils/Limelight.java:59]: @knightss27

File: src/main/java/org/team1540/robot2022/utils/Limelight.java (hunk 2 above)
Comment (natesales) [reply src/main/java/org/team1540/robot2022/utils/Limelight.java:86]: @knightss27 Is this ok to remove?

File: src/main/java/org/team1540/robot2022/utils/Limelight.java (hunk 1 above)
Comment (knightss27) [reply src/main/java/org/team1540/robot2022/utils/Limelight.java:59]: It's meters to inches conversion

**Review by rutmanz — APPROVED** [review]

**Pull request merged**